bundles/AdminBundle/Resources/public/js/pimcore/object/tags/video.js

```diff
@@ -211,9 +211,11 @@ pimcore.object.tags.video = Class.create(pimcore.object.tags.abstract, {
     },
 
     empty: function () {
+        let allowedTypes = this.data.allowedTypes;
         this.data = {
             type: "asset",
```

Comment:
the actual default currently is not `asset`, it shows an empty path field
![image](https://user-images.githubusercontent.com/6014195/188139254-de81b18d-7e3b-430f-a0fc-b529f24df637.png)

https://github.com/pimcore/pimcore/blob/a0dc084959c787a1d05ed942a5dc7b623f3be6cd/bundles/AdminBundle/Resources/public/js/pimcore/object/tags/video.js#L24 

so either we change to default "asset" on init as well or remove from there to keep consistency

Reply:
I would kick it since it is possible now that asset type is not allowed.

Reply:
```suggestion
            type: "",
```